js/app.js

```diff
@@ -133,6 +129,41 @@ function renderChart() {
     });
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////////////////////////////////
+
+// for lab 13:
+// Steps for local storage:
+// 	1. Setting items
+// 		a. JSON.stringify()  - turns javascript object into JSON object
+// 		b. localStorage.setItem("key", "value")
+// 	2. Getting items
+// 		a. localStorage.getItem("key")
+//      b. JSON.parse(variable)
+
+// function setAllProducts()
+function setItems(key, value){
+    // step 1a
+    let stringifyVersion = JSON.stringify(value);
+    // step 1b
+    localStorage.setItem(key, stringifyVersion);
+}
+
+// function getAllProducts()
+function getItems(key) {
+    // step 2a
+    let stringifyVersion = localStorage.getItem(key);
+
+    // step 2b
+    let parsedVersion = JSON.parse(stringifyVersion);
+
+    return parsedVersion;
+}
```

Comment:
Same for this one, I think this function and the last really show a solid grasp on how and when to use functions.